cluster/gce/windows/configure.ps1

```diff
@@ -109,10 +123,13 @@ try {
     StartProcess-WriteSshKeys
   }
 
+  if (IsLoggingEnabled $kube_env) {
+    InstallAndStart-LoggingAgent
```

Comment:
@pjh if stackdriver is already installed on the image, would it be started automatically?

Reply:
Yes. https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/k8s-node-setup.psm1#L1178

@liyanhui1228 could you update the comment above that function to state that even if Stackdriver is already installed, the function will still [re]start the service?

Reply:
Done.